README.md

```diff
@@ -22,17 +41,39 @@ When using `o-footer` in silent mode, the `oFooter` mixin can be used to output
 @include oFooter($simple: true);
 ```
 
+### JavaScript
+
+Unless you're using the Build Service no JS will run automatically.
+You must either construct an `o-footer` object or fire the `o.DOMContentLoaded` event, which oFooter listens for.
+
+#### Constructing an o-footer
+
+```js
+const oFooter = require('o-footer');
 
-## Upgrading from 4.x.x to 5.x.x
+const ofooter = new oFooter();
+```
+
+#### Firing an oDomContentLoaded event
+
+```js
+document.addEventListener('DOMContentLoaded', function() {
+	document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
+});
+```
```

Comment:
Would it be worth adding a note that this only needs to be done once for all Origami components?